app/src/main/java/com/mlinde/marvelcomicsapp/comicsList/ComicsListFragment.kt

```diff
@@ -65,8 +70,7 @@ class ComicsListFragment : Fragment() {
     }
 
     private fun onClick(comicBook: ComicBook){
-        val intent = Intent(requireContext(), DetailsActivity::class.java)
-        intent.putExtra("comicBook", comicBook)
-        startActivity(intent)
+        val bundle = bundleOf("comicBook" to comicBook)
+        findNavController().navigate(R.id.action_comicsListFragment_to_detailsFragment, bundle)
```

Comment:
You should use directions. Because you have `navigation-safe-args-gradle-plugin` in gradle. 
For example:
```
findNavController().navigate(ComicsListFragmentDirections.action...
```
Something like that.